README.md

```diff
@@ -29,32 +29,35 @@ writes to.
 
 ### BigQuery Limitations and Splitting
 
-BigQuery has a hard limit of ten thousand columns on any single table. This library
-can take that limitation into account by splitting schemas into multiple tables. Each
-table has some common information that are duplicated in every table, and then a set
+[BigQuery has a hard limit of ten thousand columns on any single table](https://cloud.google.com/bigquery/quotas). 
+This library can take that limitation into account by splitting schemas into multiple tables,
+although so far we have been able to avoid this complication. We retain schema
+splitting support here as an option to use in the future. The option is currently disabled.
+
+When a schema is split, each
+table has some common information duplicated in every table, and then a set
 of fields that are unique to that table. The join of these tables gives the full
 set of fields available from the ping.
 
 To decide on a table split, we include the `table_group` configuration in the configuration
 file. For example, `payload/histograms` has `table_group: histograms`; this indicates that
 there will be a table outputted with just histograms.
 
-Currently, generates tables for:
-- Histograms
-- Keyed Histograms
-- Scalars
-- Keyed Scalars
-- Everything else
-
-If a single table expands beyond 9000 columns, we move the new fields to the next table.
-For example, main_histograms_1 and main_histograms_2.
+If a single table expands beyond the configured column limit, we move the new fields to the next table.
+For example, we could have main_histograms_1 and main_histograms_2.
 
-Note: Tables are only split if the `--split` parameter is provided.
+_Note_: Tables are only split if the `--split` parameter is provided, and this
+is option is not currently used for our production configuration.
 
 ## Validation
 
-A secondary use-case of these schemas is for validation. The schemas produced are guaranteed to
-be more correct, since they include explicit definitions of every metric and probe.
+When we validate pings against a schema in the data pipeline, we use the generic versions
+rather than the versions generated by this repository's machinery. While the schemas produced
+here are guaranteed to be more correct since they include explicit definitions of every metric and probe,
+we find in practice there are too many edge cases where a probe is sent with the incorrect type
+and we need to coerce it to the correct type when loading to BigQuery.
+We could still consider using the generated schemas for validation in the future, but
```

Comment:
There are also the schemas that are purposely crafted as string types, in order to coerce them as JSON fields in the BigQuery schema. This makes them impractical for validation (unless we dropped casted fields in the generated JSON Schema).